src/NationalCloud.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.DotNet.Interactive.MicrosoftGraph
+{
+    /// <summary>
+    /// Supported national clouds.
+    /// </summary>
+    public enum NationalCloud
```

Comment:
No China?

Reply:
I don't have a way to validate China national cloud, so only included the others for time being.  Can add in China if you think ok.